diff -ruN hdfeos5.orig/include/Makefile.am hdfeos5/include/Makefile.am
--- hdfeos5.orig/include/Makefile.am	2007-03-29 18:59:35.000000000 +0200
+++ hdfeos5/include/Makefile.am	2021-09-27 22:45:51.104412174 +0200
@@ -4,6 +4,4 @@
 include $(top_srcdir)/config/include.am
 
 # Headers to install
-include_HEADERS = HE5_GctpFunc.h HE5_HdfEosDef.h HE5_config.h cproj.h ease.h \
-                  isin.h proj.h tutils.h cfortHdf.h
-
+include_HEADERS = HE5_GctpFunc.h HE5_HdfEosDef.h ease.h isin.h cfortHdf.h
diff -ruN hdfeos5.orig/include/Makefile.in hdfeos5/include/Makefile.in
--- hdfeos5.orig/include/Makefile.in	2009-05-19 18:12:51.000000000 +0200
+++ hdfeos5/include/Makefile.in	2021-09-27 23:21:08.157042433 +0200
@@ -190,8 +190,7 @@
 # Boilerplate include
 
 # Headers to install
-include_HEADERS = HE5_GctpFunc.h HE5_HdfEosDef.h HE5_config.h cproj.h ease.h \
-                  isin.h proj.h tutils.h cfortHdf.h
+include_HEADERS = HE5_GctpFunc.h HE5_HdfEosDef.h ease.h isin.h cfortHdf.h
 
 all: HE5_config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-am
